monitoring/README.md

```diff
@@ -107,7 +111,7 @@ wds1-ks-transport-controller   64m
 b) ITS space:
 
 ```bash
-./configure-monitoring-its.sh 
```

Comment:
The flag `--host-context` refers to the KubeFlex hosting cluster only. Hence, for the script **configure-monitoring-wec.sh** we're using this flag instead: `--wec-cluster-context`

Reply:
Could configure-monitoring-wds.sh involve only one context, the one for accessing the WDS?

Could configure-monitoring-its.sh involve only one context, the one for accessing the ITS?

It would be easier to understand if each of these three scripts only involves one context, for accessing the one space being configured.

Reply:
For better clarity, I just updated one of the flags for these scripts. Hence, both script have the following flags:
   a) `--kubeflex-hosting-cluster-context`: context of the KubeFlex hosting cluster;
   b) `--space-name`:  KS space name, for example: either ws1 and its1.

Reply:
This particular thread is not about the naming of flags but the design of what flags are needed. The ITS and WDS script are given the hosting cluster context and the ControlPlane name, while the WEC script is only given the context for the WEC. I am asking if we can simplify the ITS and WDS scripts to that they too are just given one contex (the ITS and WDS space contexts, respectively)?